From 5244e89bfc819bebe4d8f9d39fdbe8777c554e0c Mon Sep 17 00:00:00 2001
From: dhacker29 <dhackerdvm@gmail.com>
Date: Sat, 25 Jan 2014 02:26:20 +0100
Subject: [PATCH] We need to set some of these different for falcon

So we have to comment them out to not override our
settings

Change-Id: Ieeee24b770c2028bf72bacb65bdc7495ba06821b
---
 BoardConfig.mk | 12 ++++++------
 msm8226.mk     |  2 +-
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index 0e67574..e25d94f 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -21,7 +21,7 @@ TARGET_HAS_QC_KERNEL_SOURCE := true
 # to get rid of compilation error.
 TARGET_HAVE_HDMI_OUT := false
 TARGET_USES_OVERLAY := true
-TARGET_NO_BOOTLOADER := false
+#TARGET_NO_BOOTLOADER := false
 TARGET_NO_KERNEL := false
 TARGET_NO_RADIOIMAGE := true
 TARGET_NO_RPC := true
@@ -41,11 +41,11 @@ TARGET_BOOTLOADER_BOARD_NAME := MSM8226
 
 BOARD_KERNEL_BASE        := 0x00000000
 BOARD_KERNEL_PAGESIZE    := 2048
-BOARD_KERNEL_TAGS_OFFSET := 0x01E00000
+#BOARD_KERNEL_TAGS_OFFSET := 0x01E00000
 BOARD_RAMDISK_OFFSET     := 0x02000000
 
 # Enables Adreno RS driver
-OVERRIDE_RS_DRIVER := libRSDriver_adreno.so
+#OVERRIDE_RS_DRIVER := libRSDriver_adreno.so
 
 #TARGET_BOOTIMG_SIGNED := true
 
@@ -63,13 +63,13 @@ TARGET_USERIMAGES_USE_EXT4 := true
 BOARD_CACHEIMAGE_FILE_SYSTEM_TYPE := ext4
 BOARD_PERSISTIMAGE_FILE_SYSTEM_TYPE := ext4
 
-BOARD_KERNEL_CMDLINE := console=ttyHSL0,115200,n8 androidboot.console=ttyHSL0 androidboot.hardware=qcom user_debug=31 msm_rtb.filter=0x37
+#BOARD_KERNEL_CMDLINE := console=ttyHSL0,115200,n8 androidboot.console=ttyHSL0 androidboot.hardware=qcom user_debug=31 msm_rtb.filter=0x37
 BOARD_KERNEL_SEPARATED_DT := true
 
 BOARD_EGL_CFG := device/qcom/$(TARGET_BOARD_PLATFORM)/egl.cfg
 
-BOARD_BOOTIMAGE_PARTITION_SIZE := 0x01000000
-BOARD_RECOVERYIMAGE_PARTITION_SIZE := 0x01000000
+#BOARD_BOOTIMAGE_PARTITION_SIZE := 0x01000000
+#BOARD_RECOVERYIMAGE_PARTITION_SIZE := 0x01000000
 BOARD_SYSTEMIMAGE_PARTITION_SIZE := 838860800
 BOARD_USERDATAIMAGE_PARTITION_SIZE := 5066702848
 BOARD_CACHEIMAGE_PARTITION_SIZE := 33554432
diff --git a/msm8226.mk b/msm8226.mk
index a422f8e..ee0fce6 100644
--- a/msm8226.mk
+++ b/msm8226.mk
@@ -1,5 +1,5 @@
 TARGET_USES_QCOM_BSP := true
-BOARD_HAVE_QCA_NFC := true
+#BOARD_HAVE_QCA_NFC := true
 
 ifeq ($(TARGET_USES_QCOM_BSP), true)
 # Add QC Video Enhancements flag
-- 
1.8.1.2

